poppunk/2.5.0/Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM mambaorg/micromamba:0.24.0 as app
+
+# Version arguments
+# ARG variables only persist during build time
+ARG POPPUNK_VERSION="2.5.0"
+
+# build and run as root users since micromamba image has 'mambauser' set as the $USER
+USER root
+# set workdir to default for building; set to /data at the end
+WORKDIR /
+
+LABEL base.image="mambaorg/micromamba:0.24.0"
+LABEL dockerfile.version="1"
+LABEL software="PopPUNK"
+LABEL software.version=${POPPUNK_VERSION}
+LABEL description="POPulation Partitioning Using Nucleotide Kmers"
+LABEL website="https://github.com/bacpop/PopPUNK"
+LABEL license="https://github.com/bacpop/PopPUNK/blob/master/LICENSE"
+LABEL maintainer1="Curtis Kapsak"
+LABEL maintainer1.email="curtis.kapsak@theiagen.com"
```

Comment:
Would you like to add yourself here as a maintainer? `LABEL maintainer2="..."`

Reply:
Thanks @kapsakcj, but I think it is better to list just one maintainer and this is 95% your code. I am okay with not being mentioned on this one :)